src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml

```
@@ -44,7 +44,10 @@ akeneo_data_quality_insights:
             day: day
         key_indicators:
             title: Key Indicators
-            products_to_work_on: '{1}You have <button>1 product</button> to work on.|]1, 200[You have <button>{{ count }} products</button> to work on.|[200, Inf[You have <button>more than {{ count }} products</button> to work on.'
+            products: '{1}1 product|]1, 200[{{ count }} products|[200, Inf[more than {{ count }} products'
+            product_models: '{1}1 product model|]1, 200[{{ count }} product models|[200, Inf[more than {{ count }} product models'
+            entities_to_work_on: 'You have <button_a/> to work on.'
```

I am afraid without context this kind of label could be difficult to translate.
If we define a more readable "tag name" for the link it could ease the translation work. What do you think about the following suggestion?
```suggestion
            entities_to_work_on: 'You have <improvable_products_count_link/> to work on.'
            entities_to_work_on_2_kinds: 'You have <improvable_products_count_link/> and <improvable_product_models_count_link/> to work on.'
```

---

you're right this will make sense to translators.